fhir-registry/src/main/java/com/ibm/fhir/registry/FHIRRegistry.java

```diff
@@ -87,7 +100,7 @@ public boolean hasResource(String url, Class<? extends Resource> resourceType) {
      *     the latest version of a resource for the given url and resource type if exists, null otherwise
      */
     public String getLatestVersion(String url, Class<? extends Resource> resourceType) {
```

Comment:
Add documentation that says you can only call this for definitional resource types and that otherwise it will throw an IllegalArgumentException

Reply:
An `IllegalArgumentException` will **not** be thrown if the resource isn't definitional for this method. The method will simple return false.